torch/csrc/jit/vararg_functions.cpp

```diff
@@ -2,33 +2,16 @@
 
 namespace torch {
 namespace jit {
-void tensorListConstructFunc(int num_inputs, Stack& stack) {
-  const size_t stack_size = stack.size();
-  c10::List<at::Tensor> vals;
-  vals.reserve(num_inputs);
-  for (size_t i = stack_size - num_inputs; i < stack_size; ++i) {
-    vals.emplace_back(std::move(stack[i]).toTensor());
-  }
-  drop(stack, num_inputs);
-  push(stack, std::move(vals));
-}
 
-void tupleUnpackFunc(int num_outputs, Stack& stack) {
+void tupleUnpack(Stack& stack) {
```

Comment:
Is the story here that `num_outputs` was never really necessary? AFAICT the logic is the same, mod the `AT_ERROR` below.

Reply:
correct